### ISSUE TITLE
Create A Config file capable of configure everything on the project

### ISSUE THREAD

**Issue by HRNPH** [HRNPH/AIwaifu#63]
# Problem
we now have too many option to select from, It would be easier to have config file which can be edited and configure the whole project in one place

**Comment by Zinedinarnaut** [comment]
Couldn’t we use dotenv for that or just the config.json/yaml.

**Comment by HRNPH** [comment]
yep probably json file but i create an issue in case someone wanna do it

**Comment by Zinedinarnaut** [comment]
What needs to be configured?

**Comment by HRNPH** [comment]
The Config.json should be in the root of the file and contain
- The Model Selection (RVC model and TTS model)
- UseTranslation? Configuration
- Port of API, Port of Client
